sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -42,19 +54,28 @@ enum
 typedef struct
 {
   /**
-   * The module name (if a module identity is used).
+   * The module_id represents the module name (if a module identity is used).
```

Comment:
Can we backtick all of these?
```suggestion
   * The `module_id` represents the module name (if a module identity is used).
```

Reply:
Yeah, they should all be updated.